src/numJewelsInStones.js

```diff
@@ -28,7 +28,16 @@
  * @return {number}
  */
 function numJewelsInStones(jewels, stones) {
-  // write code here
+  const normalizedStones = stones.replace(/[^a-zA-Z ]/g, '');
```

Comment:
Если ты хочешь убрать все кроме букв, то второй аргумент опционален
```suggestion
  const normalizedStones = stones.replace(/[^a-zA-Z ]/g);
```